README.md

```diff
@@ -10,8 +10,9 @@ Overall, this project utilizes ApostropheCMS as a headless backend with Astro as
 ## Using this project
 
 ### Prerequisites
-- Node.js v20 or later
-- MongoDB v6.0 or later running on a local server or access to Atlas. See the [ApostropheCMS documentation](https://docs.apostrophecms.org/guide/development-setup.html) for set-up details.
+- Node.js v18 or later
+- MongoDB v6.0 or later (local server or Atlas). See the [ApostropheCMS documentation](https://docs.apostrophecms.org/guide/development-setup.html) for setup.
+- Windows users: We recommend using Windows Subsystem for Linux 2 (WSL2) for the best development experience. This ensures consistent behavior with our image processing tools and file system operations. [Learn more about setting up WSL2](https://learn.microsoft.com/en-us/windows/wsl/install).
```

Comment:
Isn't it a requirement, not a recommendation? It's not going to work with alternative nodejs setups on Windows.

Reply:
Fixed. Trying not to be too scary.